semanticdb/scalac/library/src/main/scala/scala/meta/internal/semanticdb/scalac/SemanticdbOps.scala

```diff
@@ -19,5 +18,4 @@ trait SemanticdbOps
     with TypeOps {
   val global: Global
   var config: SemanticdbConfig = SemanticdbConfig.default
-  val semanticdbIndex: SemanticdbIndex = new SemanticdbIndex
 }
```

Comment:
Am I right in thinking that you didn't plan to address https://github.com/scalameta/scalameta/issues/1671 in this pull request?

Reply:
Correct, this PR is already large enough and the required changes in metac are mostly unrelated.